src/gwt/src/org/rstudio/core/client/command/KeyboardShortcut.java

```diff
@@ -78,12 +78,12 @@ public int hashCode()
    }
 
    @Override
-   public String toString()
+   public String toString() //TODO: i18n?
```

Comment:
WIP on shortcut localization? Again, prefer this PR contain no code changes, just the French property files.